tests/storage/export.go

```diff
@@ -1432,6 +1435,303 @@ var _ = SIGDescribe("Export", func() {
+			vmeName = vmExport.Name
+			// Run vmexport
+			By("Running vmexport command")
+			virtctlCmd := clientcmd.NewRepeatableVirtctlCommand(commandName, "delete", vmeName, "--namespace", util.NamespaceTestDefault)
+			err = virtctlCmd()
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("Delete fails when vmexport doesn't exist", func() {
+			// Run vmexport
+			By("Running vmexport command")
+			virtctlCmd := clientcmd.NewRepeatableVirtctlCommand(commandName, "delete", vmeName, "--namespace", util.NamespaceTestDefault)
+			err = virtctlCmd()
+			Expect(err).To(HaveOccurred())
+			errString := fmt.Sprintf("VirtualMachineExport '%s/%s' does not exist", util.NamespaceTestDefault, vmeName)
+			Expect(err.Error()).Should(Equal(errString))
+		})
+
+		It("Download succeeds creating and downloading a vmexport using PVC source", func() {
+			if !checks.IsOpenShift() {
+				Skip("Not on openshift")
+			}
+
+			// Create populated PVC
+			pvc, _ := populateKubeVirtContent(sc, k8sv1.PersistentVolumeFilesystem)
+			// Run vmexport
+			By("Running vmexport command")
+			virtctlCmd := clientcmd.NewRepeatableVirtctlCommand(commandName,
+				"download",
+				vmeName,
+				"--pvc", pvc.Name,
+				"--output", defaultOutput,
+				"--volume", pvc.Name,
+				"--insecure",
+				"--namespace", util.NamespaceTestDefault)
+
+			err = virtctlCmd()
+			Expect(err).ToNot(HaveOccurred())
+			_, err = os.Stat(defaultOutput)
+			Expect(err).ToNot(HaveOccurred())
+			err = os.Remove(defaultOutput)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("Download succeeds creating and downloading a vmexport using Snapshot source", func() {
+			if !checks.IsOpenShift() {
+				Skip("Not on openshift")
+			}
+			sc, exists := libstorage.GetSnapshotStorageClass()
+			if !exists {
+				Skip("Skip test when storage with snapshot is not present")
+			}
+
+			// Create a populated Snapshot
+			blankDv := libstorage.NewBlankDataVolume(util.NamespaceTestDefault, sc, "64Mi", k8sv1.ReadWriteOnce, k8sv1.PersistentVolumeFilesystem)
+			vm := tests.NewRandomVMWithDataVolumeAndUserDataInStorageClass(
+				cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskCirros),
+				util.NamespaceTestDefault,
+				bashHelloScript,
+				sc)
+			libstorage.AddDataVolumeTemplate(vm, blankDv)
+			addDataVolumeDisk(vm, "blankdisk", blankDv.Name)
+			vm = createVM(vm)
+			snapshot := createAndVerifyVMSnapshot(vm)
+			Expect(snapshot).ToNot(BeNil())
+			defer deleteSnapshot(snapshot)
+
+			// Run vmexport
+			By("Running vmexport command")
+			virtctlCmd := clientcmd.NewRepeatableVirtctlCommand(commandName,
+				"download",
+				vmeName,
+				"--snapshot", snapshot.Name,
+				"--output", defaultOutput,
+				"--volume", snapshot.Name,
+				"--insecure",
+				"--namespace", util.NamespaceTestDefault)
+
+			err = virtctlCmd()
+			Expect(err).ToNot(HaveOccurred())
+			_, err = os.Stat(defaultOutput)
+			Expect(err).ToNot(HaveOccurred())
+			err = os.Remove(defaultOutput)
```

Comment:
Can you put this in an `AfterEach` if the stat fails, the remove is never called.